Altis_Life.Altis/core/functions/fn_startLoadout.sqf

```diff
@@ -0,0 +1,81 @@
+#include "..\..\script_macros.hpp"
+/*
+    File: fn_startLoadout.sqf
+    Author: Casperento
+
+    Description:
+    Loads a custom loadout on player when he got a new life
+*/
+private _pCloth = M_CONFIG(getArray,"Loadouts",str(playerSide),"cloth");
+private _pVest = M_CONFIG(getArray,"Loadouts",str(playerSide),"vest");
+private _pBackpack = M_CONFIG(getArray,"Loadouts",str(playerSide),"backpack");
+private _pWeapon = M_CONFIG(getArray,"Loadouts",str(playerSide),"weapon");
+private _pMagazines = M_CONFIG(getArray,"Loadouts",str(playerSide),"mags");
+private _pItems = M_CONFIG(getArray,"Loadouts",str(playerSide),"items");
+private _linkedItems = M_CONFIG(getArray,"Loadouts",str(playerSide),"linkedItems");
+
+if !(_pCloth isEqualTo []) then {
+    if (playerSide isEqualTo civilian) then {
+        _pCloth = (selectRandom _pCloth);
+        if (!(_x isEqualTo []) && !((_pCloth select 0) isEqualTo "") && ([(_pCloth select 1)] call life_fnc_levelCheck)) then {
```

Comment:
For these longer if statements, some short-circuit conditions would be better. EX:
`if (con1 && { con2  && { con3 }}) then { codeHere };`